assets/js/modules/analytics-4/datastore/properties.js

```diff
@@ -287,8 +286,10 @@ const baseActions = {
 	 */
 	*matchAndSelectProperty( accountID, fallbackPropertyID = '' ) {
 		yield {
-			type: START_MATCHING_PROPERTY,
-			payload: {},
+			type: FINISH_MATCHING_PROPERTY,
+			payload: {
+				finished: false,
+			},
```

Comment:
This is semantically a bit odd. Why don't we keep the `START_MATCHING_PROPERTY` and `STOP_MATCHING_PROPERTY` as it was before? I don't think sending a `FINISH_MATCHING_PROPERTY` action before and after the action makes sense. Also particularly since it was like that before, I don't see the value in changing it since it doesn't really affect the underlying logic.